common/types.go

```diff
@@ -22,8 +22,9 @@ const (
 	DaemonOkResp = "I'm a little Webhook, short and stout!"
 )
 
-// UpRequest is the body of a up request to the daemon.
-type UpRequest struct {
-	Stream bool   `json:"stream"`
-	Repo   string `json:"repo"`
+// DaemonRequest is the configurable body of a request to the daemon.
+type DaemonRequest struct {
+	Stream    bool   `json:"stream"`
+	Repo      string `json:"repo,omitempty"`
+	Container string `json:"container,omitempty"`
```

Comment:
As handy as this JSON language is, it feels like a smell to me

Reply:
hm what do you mean?

Reply:
It just feels like it got hacked onto the language. Its very handy, but something feels strange about it. I'm just murmuring out loud, ignore me :+1:

Reply:
true, functionality-wise I think it's quite nice though 😛